src/flip.sol

```diff
@@ -111,9 +143,17 @@ contract Flipper is LibNote {
         require(kicks < uint(-1), "Flipper/overflow");
         id = ++kicks;
 
-        bids[id].bid = bid;
+        (PipLike pip,) = spot.ilks(ilk);
+
+        // Need to be whitelisted to make this call
```

Comment:
If we don't want to whitelist the Flipper to cal the OSM, we could instead do math based on `spot` in the Vat and the `mat` value.